bigbio/biodatasets/genia_ptm_event_corpus/genia_ptm_event_corpus.py

```diff
@@ -0,0 +1,196 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Post-translational-modiﬁcations (PTM), amino acid modiﬁcations of proteins after translation, are one of the posterior
+processes of protein biosynthesis for many proteins, and they are critical for determining protein function such as its
+activity state, localization, turnover and interactions with other biomolecules. While there have been many studies of
+information extraction targeting individual PTM types, there was until recently little effort to address extraction of
+multiple PTM types at once in a unified framework.
+"""
+
+import os
+from pathlib import Path
+from typing import List, Tuple, Dict
+
+import datasets
+from bigbio.utils import parsing, schemas
+from bigbio.utils.configs import BigBioConfig
+from bigbio.utils.constants import Tasks
+
+_LOCAL = False
+_CITATION = """\
+@inproceedings{ohta-etal-2010-event,
+    title = "Event Extraction for Post-Translational Modifications",
+    author = "Ohta, Tomoko  and
+      Pyysalo, Sampo  and
+      Miwa, Makoto  and
+      Kim, Jin-Dong  and
+      Tsujii, Jun{'}ichi",
+    booktitle = "Proceedings of the 2010 Workshop on Biomedical Natural Language Processing",
+    month = jul,
+    year = "2010",
+    address = "Uppsala, Sweden",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/W10-1903",
+    pages = "19--27",
+}
+"""
+
+_DATASETNAME = "genia_ptm_event_corpus"
+
+_DESCRIPTION = """\
+Post-translational-modiﬁcations (PTM), amino acid modiﬁcations of proteins after translation, are one of the posterior
+processes of protein biosynthesis for many proteins, and they are critical for determining protein function such as its
+activity state, localization, turnover and interactions with other biomolecules. While there have been many studies of
+information extraction targeting individual PTM types, there was until recently little effort to address extraction of
+multiple PTM types at once in a unified framework.
+"""
+
+_HOMEPAGE = "http://www.geniaproject.org/other-corpora/ptm-event-corpus"
+
+_LICENSE = "GENIA Project License for Annotated Corpora"
+
+_URLS = {
+    _DATASETNAME: "http://www.geniaproject.org/other-corpora/ptm-event-corpus/post-translational_modifications_training_data.tar.gz?attredirects=0&d=1",
+}
+
+_SUPPORTED_TASKS = [Tasks.EVENT_EXTRACTION]
```

Comment:
I noticed this corpus also supports entities and coreferences, would you mind adding `Tasks.NAMED_ENTITY_RECOGNITION, Tasks.COREFERENCE_RESOLUTION` to the list?